packages/framework/src/Form/Admin/Payment/PaymentFormType.php

```diff
@@ -75,15 +86,39 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'constraints' => [
                     new Constraints\NotBlank(['message' => 'Please enter VAT rate']),
                 ],
+                'label' => t('VAT'),
+            ]);
+
+        $builderAdditionalInformationGroup = $builder->create('additional_information', GroupType::class, [
+            'label' => t('Additional information'),
+        ]);
+
+        $builderAdditionalInformationGroup
+            ->add('transports', ChoiceType::class, [
+                'required' => false,
+                'choices' => $this->transportFacade->getAll(),
+                'choice_label' => 'name',
+                'choice_value' => 'id',
+                'multiple' => true,
+                'expanded' => true,
+                'empty_message' => t('You have to create some shipping first.'),
             ])
             ->add('description', LocalizedType::class, [
                 'required' => false,
                 'entry_type' => TextareaType::class,
+                'label' => t('Description'),
             ])
             ->add('instructions', LocalizedType::class, [
                 'required' => false,
                 'entry_type' => CKEditorType::class,
-            ])
+                'label' => t('Instructions'),
+            ]);
+
+        $builderImageGroup = $builder->create('image', GroupType::class, [
+            'label' => t('Image'),
+            'is_group_container_to_render_as_the_last_one' => true,
```

Comment:
it is not used in final state, so i deleted it from commit where it was added.

Reply:
Now I'm little confused, because in documentation you say:

'Right now we need to tell form type which GroupType is the last onto render form correctly, just remember to add this option if it is last group of fields.'

And you do not use it in Payment and Transport forms, but it is still in use in many others. Which way is right? I looked in the administration and Payment form looks good to me, is there need for this option anymore?